tests/clone_test.go

```diff
@@ -696,8 +699,23 @@ var _ = Describe("[Serial]VirtualMachineClone Tests", Serial, func() {
 						libinstancetype.WaitForVMInstanceTypeRevisionNames(sourceVM.Name, virtClient)
 
 						if toRunSourceVM {
-							By("Starting the VM and expecting it to run")
-							sourceVM = RunVMAndExpectLaunchWithRunStrategy(virtClient, sourceVM, virtv1.RunStrategyAlways)
+							sourceVM.Spec.RunStrategy = ptr.To(virtv1.RunStrategyAlways)
+							patchSet := patch.New()
+							patchSet.AddOption(
+								patch.WithAdd("/spec/runStrategy", virtv1.RunStrategyAlways),
+								patch.WithAdd("/spec/running", nil),
+							)
+							payload, err := patchSet.GeneratePayload()
+							Expect(err).ToNot(HaveOccurred())
+							sourceVM, err = virtClient.VirtualMachine(sourceVM.Namespace).Patch(context.Background(), sourceVM.Name, types.JSONPatchType, payload, v1.PatchOptions{})
+							Expect(err).ToNot(HaveOccurred())
+
+							By("VM has the running condition")
+							Eventually(func() bool {
```

Comment:
can use:
Eventually(ThisVM(vm), 360*time.Second, 1*time.Second).Should(BeReady())

Reply:
Oh nice i didn't know about these functions :)

Done